### PR TITLE
IGAPP-935: Prefix yarn before Commands

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

This will fix running commands on windows. @Cmd190 please also test this.

### CODE REVIEW

File: native/package.json [hunk 1]
```diff
@@ -5,25 +5,25 @@
   "private": true,
   "scripts": {
     "start": "yarn start:integreat-test-cms",
-    "start:integreat-test-cms": "./run packager integreat-test-cms",
-    "start:integreat": "./run packager integreat",
-    "start:integreat-floss": "./run packager integreat-floss",
-    "start:malte": "./run packager malte",
-    "start:aschaffenburg": "./run packager aschaffenburg",
-    "start:integreat-e2e": "./run packager integreat-e2e",
+    "start:integreat-test-cms": "node ./run packager integreat-test-cms",
```
Comment (klinzo) [inline native/package.json:8]: Why node and not yarn?

File: native/run [hunk 2]
```diff
@@ -28,7 +28,7 @@ program
       .replaceAll('\n', ' ')
```
Comment (Cmd190) [inline native/run:28]: I get an error while executing yarn android:
PS C:\Projects\Integreat\integreat-app\native> yarn android
yarn run v1.22.10
$ yarn android:integreat-test-cms
$ node ./run android integreat-test-cms
C:\Projects\Integreat\integreat-app\native\run:28
      .replaceAll('\n', ' ')
       ^

TypeError: execSync(...).toString(...).replaceAll is not a function
    at Command.<anonymous> (C:\Projects\Integreat\integreat-app\native\run:28:8)
    at Command.listener [as _actionHandler] (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:488:17)
    at C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1227:65
    at Command._chainOrCall (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1144:12)
    at Command._parseCommand (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1227:27)
    at Command._dispatchSubcommand (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1050:25)
    at Command._parseCommand (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1193:19)
    at Command.parse (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:897:10)
    at Object.<anonymous> (C:\Projects\Integreat\integreat-app\native\run:34:9)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
error Command failed with exit code 1.

Comment (klinzo) [reply native/run:28]: @Cmd190 Can you try replacing it with `.replace(/\n/g, ' ')`?

Comment (klinzo) [reply native/run:28]: What is your node version?

Comment (Cmd190) [reply native/run:28]: @klinzo Node Version v14.17.3

Comment (Cmd190) [reply native/run:28]: > @Cmd190 Can you try replacing it with `.replace(/\n/g, ' ')`?

The error is still the same

Comment (klinzo) [reply native/run:28]: You mean it is now `....replace(...)` is not a function instead of `....replaceAll(..)`?

Comment (Cmd190) [reply native/run:28]: I oversaw the change from `replaceAll` to `replace`  and just changed the Regex. After using only `replace` is build correctly

Comment (Cmd190) [reply native/run:28]: Same goes for yarn start

Comment (klinzo) [reply native/run:28]: Perfect, if you want you can commit your change here and start your branch from there.

**Review by f1sh1918 — APPROVED** [review]
works on macOS

**Comment by Taggotty** [comment]
Again updated this branch as we found something else not working. `yarn cross-env ${buildConfig}` seems to have swallowed the `react-native run-android` command. Setting the env variables beforehand for the whole shell  seems to have finally fixed the issue. @Cmd190  please check this out again.

**Pull request merged**